asf_heat_pump_suitability/analysis/evaluate_hp_suitability_outputs/pandera_eval_hps_outputs.py

```diff
@@ -15,11 +15,27 @@
 import pandera.polars as pa
 import json
 import config.dq_config as cfg
+import argparse
 from pandera.polars import Check, Column, DataFrameSchema
 from asf_heat_pump_suitability.analysis.hn_zones.hnz_utils.log_utils import (
     setup_logging_and_file_path,
 )
 
+# ------------------------------------------------------------------------------
+# Argument parsing for data path
+# ------------------------------------------------------------------------------
+parser = argparse.ArgumentParser(
+    description="data quality checks for heat pump suitability data."
+)
+parser.add_argument(
+    "--data-path",
+    type=str,
+    default=cfg.DATA_PATH,
+    help="Path to the data CSV file (local or S3 URI).",
```

Comment:
```suggestion
    help="Path to the heat pump suitability per LSOA data CSV file (local or S3 URI).",
```
I would put something like this for clarity. Also, you can put in the docstring at the top of the file that this script takes the heat pump suitability per LSOA data which is an output from `asf_heat_pump_suitability/pipeline/run_scripts/run_calculate_suitability.py`